lib/plugin.js

```diff
@@ -329,6 +363,8 @@ async function _processPreUpdate (options) {
   const changes = DeepDiff.getDiff({ old, current }, schema, options)
   const docMapped = _mappedFieldsLog({ changes, method, module, documentNumber, user })
 
+  options.connectionUri = this.model.db._connectionString
```

Comment:
maybe extract this as well in a common function so that we have this setting only in one place

e.g. `getConnectionUri()`